src/components/Story.js

```diff
@@ -17,7 +18,7 @@ const Story = () => {
   // HACK: I should figure out a way to use xstate to migrate from loading to ready
   //  but b/c the async/await nature of the callbacks with Holistic, I'm leaving this hack in for now.
   const [loading, setLoading] = useState(true);
-  const [poseData, setPoseData] = useSetState({});
+  const [poseData, setPoseData] = useState({}); // doesn't momeize
```

Comment:
memoize :)

Reply:
also, if we're not using the `useSetState` hook, we should remove its import statement, too, right?